status/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from django.template import RequestContext
+
+from problem.models import Submission, SubmissionDetail
+from index.views import custom_proc
+# Create your views here.
+
+
+def status(request):
+    submission_details = SubmissionDetail.objects.all()
```

Comment:
It's not proper to query all submissions since there will be up to 1 million submissions or more.
I prefer query last 50 submissions only.